rmw_opendds_cpp/src/rmw_node.cpp

```diff
@@ -30,47 +30,39 @@ rmw_create_node(
   size_t domain_id,
   bool localhost_only)
 {
-  RCUTILS_CHECK_ARGUMENT_FOR_NULL(context, NULL);
-  RMW_CHECK_TYPE_IDENTIFIERS_MATCH(
-    init context,
-    context->implementation_identifier,
-    opendds_identifier,
-    // TODO(wjwwood): replace this with RMW_RET_INCORRECT_RMW_IMPLEMENTATION when refactored
-    return NULL);
-  return create_node(opendds_identifier, name, namespace_, domain_id);
+  (void)domain_id;
+  (void)localhost_only;
+  RMW_CHECK_FOR_NULL_WITH_MSG(context, "context is null", NULL);
+  if (!check_impl_id(context->implementation_identifier)) {
+    return NULL;
```

Comment:
line 35 prints an error. Shouldn't this also print an error?

Reply:
It's done in check_impl_id().